Desktop/modules/ribbonmodel.cpp

```diff
@@ -110,6 +111,27 @@ void RibbonModel::loadModules(std::vector<std::string> commonModulesToLoad, std:
 	}
 }
 
+// So we reload the special module to ensure translations can be hot updated.
+void RibbonModel::reloadModulesMenu()
+{
+    std::vector<std::string> allButtons;
+
+    for (const auto& rowButtons : _buttonNames)
+    {
+        for (const auto& button : rowButtons)
+        {
+            allButtons.push_back(button);
+        }
+    }
+
+    for (const auto& button : allButtons)
+    {
+        removeRibbonButtonModel(button);
+    }
+
+    loadModules();
+}
+
 void RibbonModel::addRibbonButtonModelFromDynamicModule(Modules::DynamicModule * module)
```

Comment:
I don't think this is the right way to do this. The loadModules(I) will recreate the modules, make some memory leakage (_entriesInsert, _entriesDelete, etc...), this is quite dangerous I think.
I don't understand why the 'special ribbon entries' are not translated, and the module ribbon buttons are. I remember that @JorisGoosen knew why. Maybe it is better to wait for his return of his holidays.